include/forms.php

```diff
@@ -2,12 +2,11 @@
 /**
  * Library for doing repetetive forms stuff
  */
-function make_select($name, $selected, $ar, $jscallback="", $cssclass=''){
```

Comment:
Ah crap, I think that was accidental. Maybe the extra argument was added after the changes I made? Maybe I had a stale branch.